docs/kinto-plugins-admin.rst

```diff
@@ -0,0 +1,42 @@
+.. _kinto-plugins-admin:
+
+Kinto-Plugins-Admin
+###################
+
+It is a Web admin UI to manage data from a Kinto server.
+
+How  to enable it?
+==================
+
+It is possible to extend the default Kinto behaviors by using “plugins”.
+
+Plugins are loaded at startup can be specified in the settings(kinto/config/kinto.ini), as a list of Python modules.
+Add the following lines of code in the *kinto.ini* file.
+
+::
+
+    kinto.includes = kinto.plugins.default_bucket
+                     kinto.plugins.history
+                     kinto-attachment
+                     custom-myplugin
```

Comment:
The plugin we are talking about for the kinto-admin is `kinto.plugins.admin` Maybe we can add it in the list.